### ISSUE TITLE
show progress report of osrm-extract

### ISSUE THREAD

**Issue by MichalPP** [Project-OSRM/osrm-backend#3891]
it would be nice to have a progress report of osrm-extract steps. in my case having any progress output of `Computing edge weights` step would help. something like `done 123 edges of 234 total` every 30 minutes or so. 

also the output `Computing edge weights    ... ok, after 10607.4s` could add total number of edges computed. is it the same number as in `[info] Writing used edges       ... ok, after 0.023866sProcessed 468917 edges`?

**Comment by emiltin** [comment]
duplicate of #419

**Comment by MichalPP** [comment]
@emiltin not really, maybe I should rephrase it to be "give `Computing edge weights` step the same percetnage progress report that other steps have" (since this step is the longest I have)

**Comment by daniel-j-h** [comment]
@MichalPP feel free to give it a try. Here is the location where edge weights are being computed, and the utility class for printing progress:

- https://github.com/Project-OSRM/osrm-backend/blob/273fd689ce531ff7eea0b671dc64435ba3b71c6f/src/extractor/extraction_containers.cpp#L349-L438
- https://github.com/Project-OSRM/osrm-backend/blob/273fd689ce531ff7eea0b671dc64435ba3b71c6f/include/util/percent.hpp#L20-L58
